diagnostic.md

```diff
@@ -45,21 +55,27 @@ weather === "sunny"
 
 What do these expressions evaluate to?  Explain your answers.
 
+let weather will show up as 'undefined' because it is simply announcing the variable, but does not define it.
+weather = "sunny" assigns a value to the variable (being sunny) because of the '=' sign.
+weather === "sunny" will state that the boolean value of the variable is true because of the "===".
+
 
 ## Question 5
 
 ```js
 //We'll learn about require later in the course
 const ask = require('../lib/ask.js')
```

Comment:
You have the right idea, just make sure that you're using the `ask` variable in your code